### PR TITLE
[ADD] accounting: add a new doc about Invoice Online Payment

### PR DESCRIPTION
**Pull request by jcs-odoo**
- This doc replaces the one about Credit Cards.

  - removed accounting/receivables/customer_payments/credit_cards.rst 

- The content related to Credit Cards has been moved to general/payment_acquirers/payment_acquirers.rst
- redirection:

  - accounting/receivables/customer_payments/credit_cards.rst accounting/receivables/customer_payments/online_payment.rst

### CODE REVIEW

**Comment by jcs-odoo** [comment]
This branch is based on a commit still not merged.
I leave this PR in draft for reviewing but I'll have to rebase it once #676 is merged.

File: accounting/receivables/customer_payments/online_payment.rst [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+   Payment Acquirer activated, but you still have to fill out the payment details.
+
+To activate the Invoice Online Payment, go to :menuselection:`Accounting --> Configuration -->
+Customer Payments`, enable **Invoice Online Payment**, and click on *Save*.
+
+Customer Portal
+===============
+
+After issuing the invoice, click on *Send & Print* and send the invoice by email to the customer.
+They will receive an email with a link to the see the invoice on their **Customer Portal**.
```
Comment (kbr-odoo) [inline accounting/receivables/customer_payments/online_payment.rst:28]:
```suggestion
They will receive an email with a link that redirects them to the invoice on their **Customer Portal**.
```
They will receive an email with a link to see the invoice on their **Customer Portal**.

File: accounting/receivables/customer_payments/online_payment.rst [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+.. image:: media/online-payment-view-invoice.png
+   :align: center
+   :alt: Email with a link to view the invoice online on the Customer Portal.
+
+They can choose which Payment Acquirer to use to pay their invoice by clicking on *Pay Now*.
```
Comment (kbr-odoo) [inline accounting/receivables/customer_payments/online_payment.rst:34]:
```suggestion
They can choose which Payment Acquirer to use by clicking on *Pay Now*.
```

File: accounting/receivables/customer_payments/online_payment.rst [hunk 3]
```diff
@@ -0,0 +1,42 @@
+======================
+Invoice Online Payment
+======================
+
+To make it more convenient for your customers to pay the invoices you issue, you can activate the
+**Invoice Online Payment** feature, which adds a *Pay Now* button on their **Customer Portal**.
```
Comment (kbr-odoo) [inline accounting/receivables/customer_payments/online_payment.rst:6]: "Allow your customer to just not see, but also pay their invoices through their Customer Portal. Offering extra convenient payment options, you show flexibility and appeal to a wider audience."

File: accounting/receivables/customer_payments/online_payment.rst [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+
+.. image:: media/online-payment-view-invoice.png
+   :align: center
+   :alt: Email with a link to view the invoice online on the Customer Portal.
+
+They can choose which Payment Acquirer to use to pay their invoice by clicking on *Pay Now*.
+
+.. image:: media/online-payment-pay-now.png
+   :align: center
+   :alt: *Pay Now* the invoice in the Customer Portal.
```
Comment (kbr-odoo) [inline accounting/receivables/customer_payments/online_payment.rst:38]:
```suggestion
   :alt: View of an invoice and the option to *Pay Now* through the Customer Portal.
```

File: accounting/receivables/customer_payments/online_payment.rst (hunk 3 above)
Comment (jcs-odoo) [reply accounting/receivables/customer_payments/online_payment.rst:6]: Thanks, I added a sentence based on your suggestion :)

**Comment by jcs-odoo** [comment]
Thank you @kbr-odoo  :)

**Pull request merged**